extensions/amp-a4a/rtc-documentation.md

```diff
@@ -13,6 +13,8 @@ Remote HTML support was added for Delayed Fetch to support first-party cookie ta
 
 ## Overview
 
+RTC works by sending HTTP requests to URLs that a publisher specifies for each ad slot. The servers that receive these RTC requests respond with a JSON object that the associated AMP Fast Fetch implementation can use for various purposes, for instance the DoubleClick Fast Fetch implementation takes the data and adds it to the ad request to DFP as targeting parameters.
```

Comment:
sending XHR CORS requests... response with appropriate CORS headers and the body is a JSON object that...